SpatialGDK/Source/SpatialGDK/Public/Schema/RPCPayload.h

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/Component.h"
+#include "SpatialConstants.h"
+#include "Utils/SchemaUtils.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+namespace SpatialGDK
+{
+
+struct RPCPayload
+{
+	RPCPayload() = delete;
+
+	RPCPayload(uint32 InOffset, uint32 InIndex, const TArray<uint8>& Data) : Offset(InOffset), Index(InIndex), PayloadData(Data)
```

Comment:
I think I'm of the mind that this constructor should be deleted. I don't see it being used anywhere and I think I would prefer always moving `Data` into an RPCPayload object rather than copy. Not having this constructor will force people who use RPCPayload to always move `Data`.

Reply:
Removed.